<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CaveWindow</class>
 <widget class="QMainWindow" name="CaveWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>613</width>
    <height>496</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CaveWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_11">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="LFP">
            <property name="text">
             <string>File Path</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LFilePath">
            <property name="text">
             <string>home_directory</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="LMN">
            <property name="text">
             <string>File Name</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LCaveName">
            <property name="text">
             <string>simple_cave</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <item>
               <widget class="QLabel" name="LColumnsNumber">
                <property name="text">
                 <string>Columns Number</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SBColumnsNumber">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>50</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QLabel" name="LRowsNumber">
                <property name="text">
                 <string>Rows Number</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QSpinBox" name="SBRowsNumber">
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
                <property name="minimum">
                 <number>1</number>
                </property>
                <property name="maximum">
                 <number>50</number>
                </property>
                <property name="value">
                 <number>10</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="PBCreate">
            <property name="text">
             <string>Create</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PBNexStep">
            <property name="text">
             <string>Nex Step</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LTimeBetweenSteps">
            <property name="text">
             <string>Time between steps in miliseconds</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBTimeStep">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="minimum">
             <number>25</number>
            </property>
            <property name="maximum">
             <number>1000</number>
            </property>
            <property name="singleStep">
             <number>25</number>
            </property>
            <property name="value">
             <number>500</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PBAutoCaveFormationStart">
            <property name="text">
             <string>Start Auto Cave Formation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="PBAutoCaveFormationStop">
            <property name="text">
             <string>Stop Auto Cave Formation</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LLifeLimit">
            <property name="text">
             <string>Set Life limit 1- 7</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBLifeLimit">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>7</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LDeathLimit">
            <property name="text">
             <string>Set Death Limit</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="SBDeathLimit">
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="maximum">
             <number>7</number>
            </property>
            <property name="value">
             <number>4</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LCurrentNumber">
            <property name="text">
             <string>Current Step</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="LCurrentStepNumber">
            <property name="text">
             <string>0</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QGridLayout" name="GLCave"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>613</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad_Cave"/>
    <addaction name="actionSave_Cave"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_Cave">
   <property name="text">
    <string>Load_Cave</string>
   </property>
  </action>
  <action name="actionSave_Cave">
   <property name="text">
    <string>Save_Cave</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
